tests/test_metadata.py

```diff
@@ -0,0 +1,83 @@
+from mapreader import loader
+import pytest
+import os
+import pandas as pd
+import shutil
+
+@pytest.fixture
+def dir_name():
+    return "metadata_test_dir"
+
+#if there is extra info in the metadata
+def test_extra_metadata(dir_name):  
+    os.mkdir(dir_name)
```

Comment:
This fails if `dir_name` already exists.

Reply:
as above